c++/src/kj/compat/tls-test.c++

```diff
@@ -1055,6 +1054,60 @@ KJ_TEST("TLS receiver does not stall on hung client") {
   auto extraAcceptPromise = test.receiver->accept().then([&](auto server) {
```

Comment:
Looks like this could now use `.then(readFromClient)`

Reply:
Yep, thanks!